templates/oauth.html

```diff
@@ -0,0 +1,14 @@
+{% extends "layout.html" %}
+
+{% block content %}
+<div class="row justify-content-center">
+<div class="col-sm-6">
+    <img class="mb-4" src="https://zap.me/app/themes/zap/assets/public/images/logo-footer.svg" alt="" width="72" height="72">
+    <h1 class="h3 mb-3 font-weight-normal">Login Required: </h1>
+        {%- for message in get_flashed_messages() %}
```

Comment:
We should move this "flashed messages" block into layout.html so we can show them on ANY page if we need to

Reply:
added this but not sure if its correct.